### PR TITLE
[Merged by Bors] - [CU-2qvh2mw] Update base-ci-linix and ci-linux Docker images

### PR DESCRIPTION
**Pull request by andor0**
## Issue
https://app.clickup.com/t/2qvh2mw

Also, I updated `docker/composable-sandbox.dockerfile` and fixed clippy remarks.

### CODE REVIEW

**Comment by ComposableFiB0t** [comment]
Task linked: [CU-2qvh2mw Update base-ci-linix and ci-linux Docker images](https://app.clickup.com/t/2qvh2mw)

**Review by dzmitry-lahoda — APPROVED** [review]

**Review by dzmitry-lahoda — APPROVED** [review]

File: docker/composable-sandbox.dockerfile [hunk 1]
```diff
@@ -1,6 +1,5 @@
-FROM composablefi/ci-linux:2022-04-18 as builder
+FROM composablefi/ci-linux:2022-08-06 as builder
 
-# ISSUE: we already copied context when started build, what the heck we are roing here? it slows down builds
```
Comment (andor0) [inline docker/composable-sandbox.dockerfile:3]: we'll get
```
$ make containerize-composable-sandbox
Sending build context to Docker daemon  2.427GB
Step 1/23 : FROM composablefi/ci-linux:2022-08-06 as builder
 ---> 1a4fdb751b4f
Step 2/23 : RUN cargo +nightly build --release -p wasm-optimizer
 ---> Running in 6c40e1519b87
error: could not find `Cargo.toml` in `/` or any parent directory
The command '/bin/bash -o pipefail -c cargo +nightly build --release -p wasm-optimizer' returned a non-zero code: 101
make: *** [Makefile:107: containerize-composable-sandbox] Error 101
```
if remove 2 commands below

Comment (dzmitry-lahoda) [reply docker/composable-sandbox.dockerfile:3]: sure. please delete comment then. thank you. i was wrong. so it kind of limitation. you send context and additionally remount all of it second time by desing of docker.

**Review by dzmitry-lahoda — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -136,19 +136,19 @@ push-mmr-polkadot:
 
 containerize-ci-linux:
 	@docker build -f docker/ci-linux.dockerfile \
-		-t ${REPO}/ci-linux:2022-04-18  \
+		-t ${REPO}/ci-linux:2022-08-06  \
```
Comment (benluelo) [inline Makefile:139]: Should the ci-linux version be extracted out into a variable?

**Review by benluelo — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (andor0) [reply Makefile:139]: Yes, it will be more simple for an update.
I will do it in the next PR.

**Review by PoisonPhang — APPROVED** [review]

**Review by alexkdefi — APPROVED** [review]
👍

**Comment by alexkdefi** [comment]
bors++

**Pull request closed without merging**